### ISSUE TITLE
Please consider endpoint filtering for FTP/SFTP connectors

### ISSUE THREAD

**Issue by blandry613** [MicrosoftDocs/power-platform#1915]
We had a need for this with the standard SFTP connector. Can you please add that for consideration in future releases?

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 0ed8b795-d48b-9f40-58a0-27ce00bcde93
* Version Independent ID: 0ed8b795-d48b-9f40-58a0-27ce00bcde93
* Content: [Connector endpoint filtering (preview) - Power Platform](https://learn.microsoft.com/en-us/power-platform/admin/connector-endpoint-filtering)
* Content Source: [power-platform/admin/connector-endpoint-filtering.md](https://github.com/MicrosoftDocs/power-platform/blob/main/power-platform/admin/connector-endpoint-filtering.md)
* Service: **power-platform**
* Sub-service: **admin**
* GitHub Login: @mikferland-msft
* Microsoft Alias: **miferlan**

**Comment by pwbyrne** [comment]
Agreed.  End point filtering for FTP is of no less importance than HTTP, SQL or the others that are currently enabled.

Any connector that allows a configurable external endpoint that could be used to move data outside the tenant needs to be configurable.

**Comment by bishalgoswami** [comment]
Thanks for your comment, @blandry613. This issue has been assigned to the doc author for review.
